Add-ons/UmbracoForms/Editor/Attaching-Workflows/Workflow-Types/index.md

```diff
@@ -1,126 +1,121 @@
+- **Save as an XML file**
 
-* URL (required)
-* Method
-* XsltFile - used to transform the XML
-* Headers - map the needed files
-* User and password
+    ![Save as XML](images/save-as-an-xml-file.png)
 
-## Save as an XML file
+    Saves the result of the Form as an XML file by using XSLT. The following configuration can be set:
 
-![Save as XML](images/save-as-an-xml-file.png)
+  - Path (required) - where to save the XML file
+  - File extension (required)
+  - XsltFile - used to transform the XML
 
-This workflow will save the result of the form as an XML file by using XSLT.
-In the configuration you can configure the following settings:
+- **Save as Umbraco Content Node**
 
-* Path - where to save the XML file (required)
-* File extension (required)
-* XsltFile - used to transform the XML
+    ![Save as content node](images/save-as-content-node.png)
 
-## Save as Umbraco Content Node
+    Saves a submitted Form as a new content node. You need to choose a Document type and match the fields in the Form with the properties on the selected Document Type.
 
-![Save as content node](images/save-as-content-node.png)
+    You can also choose to set a static value to fill in the properties:
 
-This workflow gives you the option to save a forms submission as a new content node.
+    ![Save as content node](images/create-new-node.png)
 
-First of all, you need to choose a Document type and match the fields in the form with the properties on the selected Document Type. You can also choose to set a static value to fill in the properties.
+    In the example above, a Document Type called **Blogpost** is selected for creating the new Content node.
 
-![Save as content node](images/create-new-node.png)
+    The value from the **Name** field will be added as the **Node Name** property in the new Content node and the value from the **Email** field will be used as the **Content** property.
 
-In the example above, a Document Type called 'Blogpost' is selected to be used for creating the new content node. Furthermore, the value from the 'Name' field will be added as the 'Node Name' property in the new content node, and the value from the 'Email' field will be used for the 'Content' property.
+    The following configuration can be set:
 
-Other configuration:
+  - Publish - choose whether to publish the node on submission
+  - Where to save - choose a section in the content tree where this new node should be added
 
-* Publish - choose whether to publish the node on submission
-* Where to save - choose a section in the content tree where this new node should be added
+- **Send Email**
 
-## Send email
+    ![Send email](images/send-email.png)
 
-![Send email](images/send-email.png)
+    Sends the result of the Form to the specified email address. The following configuration can be set:
 
-Send the result of the form to a specified email address.
+  - Email (required)
+  - SenderEmail - also configurable in `Config/umbracosettings.config`
```

Comment:
I don't think this is the same in V9 🤔

As we are not using the .Config files no more, i think we will need to hear one of the forms people how this is done in V9.

Reply:
oh yes, Good catch.. the UI still has this path though.. 
![image](https://user-images.githubusercontent.com/82437098/123093109-8d0c2e00-d42b-11eb-84ff-d1710d04427e.png)

Reply:
Yeah i noticed that as well... let's reach out to Some of the people working with forms, they should be able to give us an answer :)